pkg/controllers/management/auth/globalroles/globalrole_handler.go

```diff
@@ -37,6 +49,7 @@ func newGlobalRoleLifecycle(management *config.ManagementContext) *globalRoleLif
 type globalRoleLifecycle struct {
 	crLister rbacv1.ClusterRoleLister
 	crClient rbacv1.ClusterRoleInterface
+	nsLister normanv1.NamespaceLister
```

Comment:
Nit: When adding a new field like this we tend to use the Wrangler caches instead of the Norman ones. If you could swap it I would appreciate it.